pkg/ccl/backupccl/backup_test.go

```diff
@@ -4004,7 +4015,8 @@ func TestRestoreAsOfSystemTime(t *testing.T) {
 	sqlDB.QueryRow(t, `SELECT cluster_logical_timestamp()`).Scan(&ts[3])
 
 	sqlDB.Exec(t, `DELETE FROM data.bank WHERE id >= $1 / 2`, numAccounts)
-	sqlDB.Exec(t, `ALTER TABLE other.sometable RENAME TO data.sometable`)
+	sqlDB.Exec(t, `CREATE TABLE data.sometable AS SELECT * FROM other.sometable`)
```

Comment:
Ah it's because we disable moving tables from database to database even if they're in the public schema now due to public schema being a regular schema. It was previously allowed as a special case but now generally disallowed in sql.

Reply:
ahaa got it, thanks!